### ISSUE TITLE
Add build instructions

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/android-components#620]
I'm getting compile errors and I'm not sure if it's because I'm building it wrong, my build env is broken, or the build is broken.

**Comment by pocmo** [comment]
It would be interesting to know what errors you see. In theory just importing it in Android Studio or using ./gradlew works without any fancy setup needed.

**Comment by mcomella** [comment]
Building samples-toolbar:
```
Caused by: java.io.IOException: Failed to find byte code for java/util/function/BiConsumer
    at com.android.build.gradle.internal.incremental.AsmUtils.lambda$static$0(AsmUtils.java:89)
    at com.android.build.gradle.internal.incremental.AsmUtils.loadClass(AsmUtils.java:307)
```

Then I tried a clean build beacause the internet suggested it: `com.android.builder.merge.DuplicateRelativeFileException: More than one file was found with OS independent path 'lib/x86/libjnidispatch.so'` Christian told me to remove a file.

Then read some more on the internet, which told me to disable instant run and it worked (here is link: https://stackoverflow.com/questions/49619910/android-compiling-error-failed-to-find-byte-code-for-java-util-function-functi)

---

Basically, build instructions are a piece of mind that an unfamiliar project you're building is doing things a standard way and that your build errors are coming from a real build error and not an initial project configuration error.

**Comment by pocmo** [comment]
Aye, the error you saw is something that I debugged too: #583. This was caused by a change that didn't clean up a local file that was hidden by .gitignore.

**Comment by mcomella** [comment]
FYI: there is a PR open in Shared-docs that could help address this: https://github.com/mozilla-mobile/shared-docs/pull/86
